sqld/proto/replication_log.proto

```diff
@@ -20,8 +20,13 @@ message Frame {
     bytes data = 1;
 }
 
+message Frames {
+    repeated Frame frames = 1;
+}
+
 service ReplicationLog {
     rpc Hello(HelloRequest) returns (HelloResponse) {}
     rpc LogEntries(LogOffset) returns (stream Frame) {}
+    rpc LogEntriesSnapshot(LogOffset) returns (Frames) {}
```

Comment:
I'm not sure if Snapshot is a good name since we're using that name already for different concept.

Reply:
Maybe something like LogEntriesNonblocking would better describe the difference from LogEntries?

Reply:
or maybe LogEntriesBatch?